### DISCUSSION TITLE
Still unable to generate images using Transformers 4.25.1+

### DISCUSSION THREAD

**Discussion by BlackWyvern** [vladmandic/sdnext#407] [Q&A]
So I'm linking this in here, [#7351](https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/7351) as I still am unable to generate images using that version of transformers in the requirement files.

If I downgrade the requirement to 4.19.2, it works fine. But, this issue is persisting across multiple different branches and generators. So, it's not a localized issue to simply A1x4

I'm out of ideas, and the few other places I've tried asking around in were met with silence.

So, as a more generalized discussion area, are there perhaps any other insights out there?

**Comment by vladmandic** [comment]
i'm surprised this is first i hear of this. to avoid noise, can you post single identical image genated using default sd 1.5 model and vae and using `transformers` 4.19.2 and 4.26.1 (default here). post both images as-is (dont edit or combine) so they can be downloaded with full metadata.

**Comment by BlackWyvern** [comment]
Absolutely.

Generating using Automatic 1111, Op-Auto is below.

1.5 ema-only safetensor from [runwayml](https://huggingface.co/runwayml/stable-diffusion-v1-5/tree/main)
On vae-ft-mse-840000-ema-pruned.ckpt from [stabilityai](https://huggingface.co/stabilityai/sd-vae-ft-mse-original/tree/main)
Launch arguments on A1x4: --xformers --medvram --autolaunch --no-half-vae
Commit hash: 22bcc7be428c94e9408f589966c2040187245d81
All LoRAs, embeddings, and Hypernets disabled.
ETA weight: 31337, CFG: 7

30 Step Euler A
Simple and complex prompts, 

Automatic 1111

4.19.2
![00180-2474789303](https://user-images.githubusercontent.com/7834910/233856854-c067f69c-44f1-4780-af67-f7f8b76b51dc.png)
![00183-2512420455](https://user-images.githubusercontent.com/7834910/233856974-771adabf-fdb3-47f6-8ee8-206191ef1466.png)


4.25.1
![00181-2474789303](https://user-images.githubusercontent.com/7834910/233856863-7746278a-68c6-400e-89c1-343c5103421c.png)
![00182-2512420455](https://user-images.githubusercontent.com/7834910/233856981-c73ef918-857c-45c4-8412-acec871f5675.png)

=================

Generating on Op-Auto
Simple prompt only

Same models and Vae weights.
100% stock install. (Haven't played with it enough.)
No launch arguments.
Commit hash 98adfb3151575f1020ecda509956ed9bef30ea86

4.19.2
```
11:08:34-883266 WARNING  Package wrong version: transformers 4.26.1 required 4.19.2
11:08:34-885244 INFO     Installing package: transformers==4.19.2
11:09:20-721182 ERROR    Error running git with args: checkout master
11:09:20-754095 ERROR    Error running git with args: pull --rebase --autostash
```

![00000-2512420455-3d rendering of a small metal cube sitting on a glass table](https://user-images.githubusercontent.com/7834910/233857420-d8bb65ea-ca91-4e77-a3a8-6425e8b17645.jpg)


4.26.1
```
11:12:45-120219 WARNING  Package wrong version: transformers 4.19.2 required 4.26.1
11:12:45-123227 INFO     Installing package: transformers==4.26.1
```

![00001-2512420455-3d rendering of a small metal cube sitting on a glass table](https://user-images.githubusercontent.com/7834910/233857561-95454cef-85c5-49d1-a92f-7dbbdcb9890c.jpg)

**Comment by vladmandic** [comment]
i've created issue for this so i don't forget :)

**Comment by BlackWyvern** [comment]
Updated the OP with default launch params for Op-Auto on 4.26.1 as well, as I initially missed that.
